eyelid-match-ops/src/primitives/yashe/conf.rs

```diff
@@ -103,6 +222,10 @@ where
     // TODO: work out how to const_assert!() this constraint.
     debug_assert!((C::T as u128) < C::modulus_as_u128());
 
+    // The lifted modulus `PolyBN::Coeff::MODULUS` must be large enough to hold
+    // `Self::Coeff::MODULUS^2 * T`, to implement `Yashe::ciphertext_mul()`.
+    debug_assert!(C::bn_modulus_as_big_uint() >= C::modulus_as_big_uint().pow(2) * C::T);
```

Comment:
Assertion that makes sure T and both moduli are the right size.

Reply:
Instead of T we need to use the degree. I adapted it. Thanks for creating this assert.

Reply:
Thanks!